client/js/lounge.js

```diff
@@ -932,13 +955,23 @@ $(function() {
 		);
 	}
 
+	function completeNicks(word) {
+		const users = chat.find(".active .users");
+
+		// Lobbies and private chats do not have an user list
+		if (!users.length) {
+			return [];
+		}
+
+		const words = users.data("nicks");
+
+		return fuzzyGrep(word, words);
+	}
+
```

Comment:
Actually, instead of duplicating this function, make it work for both with either of these solutions:

- Add a `isFuzzy` boolean to the signature, and if `true` return `fuzzyGrep(...)`, otherwise return `$.grep(...)`.
- Make this functional-like (think of `.map()` or `.filter`): add a `callback` argument and return `callback(words)` so that functions calling this can do `completeNicks(word, (words) => fuzzyGrep(word, words));` (or `$.grep` equivalent).

Or those are overkill and there is a much simpler solution, I don't know, it's pretty late here :)
@xPaw, any preference?